### PR TITLE
create recipe snippet files and add  basic styles from business ethic bca

### PR DESCRIPTION
**Pull request by ajluk**
for #3772

### CODE REVIEW

File: styles/design-settings/cardboard/parts/_table-settings.scss [hunk 1]
```diff
@@ -55,6 +55,26 @@
...
+            border-right-color: (_ref: "colorMap:::tableCellBorderColor"),
+        ),
+        LastTableDataMiddleHeaderCell: (
+            border-left-color: (_ref: "colorMap:::tableCellBorderColor"),
+        ),
+        TableDataMiddleHeaderCellOnlyChild: (
+            font-family: (_ref: "typography:::baseFont"),
+            color: (_ref: "colorMap:::tableHeaderColor"),
+            background-color: (_ref: "colorMap:::tableHeaderBgColor"),
+        ),
```
Comment (ajluk) [inline styles/design-settings/cardboard/parts/_table-settings.scss:77]: The middle header row tds shoud have the same specs as second header rows #2741

Comment (ajluk) [inline styles/design-settings/cardboard/parts/_table-settings.scss:77]: The middle header row tds should have the same specs as second header rows #2741 for default/top tiled tables.

File: styles/design-settings/cardboard/parts/_table-settings.scss [hunk 2]
```diff
@@ -136,6 +156,26 @@
...
```
Comment (ajluk) [inline styles/design-settings/cardboard/parts/_table-settings.scss:178]: See the comment from above.

File: styles/design-settings/cardboard/parts/_table-settings.scss [hunk 3]
```diff
@@ -180,6 +220,12 @@
             font-family: (_ref: "typography:::baseFont"),
             color: (_ref: "colorMap:::tableCellColor"),
         ), 
+        TableDataMiddleHeaderCellBottomBorder: (
+            font-family: (_ref: "typography:::baseFont"),
+            color: (_ref: "colorMap:::tableFirstHeaderColor"),
+            background-color: (_ref: "colorMap:::textheavyTableHeaderBgColor"),
+            border-bottom-color: (_ref: "colorMap:::textheavyTableHeaderBottomBorderColor"),
+        ),
```
Comment (ajluk) [inline styles/design-settings/cardboard/parts/_table-settings.scss:228]: Similarly the middle header rows tds in text-heavy tables should also have the same specs as second header rows specified in #2741.

File: recipes/books/bca/_config.scss [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+// "name" has different value in books from other countries (see pl-u-physics), but we need value common for all books (see mixin "modify_addFirstElementClas")
+$Config_BookCompositePages: (
+  (className: "index",        clusterBy: $CLUSTER_NONE, specialPageType: $PAGE_INDEX, name: "Index"),
+);
+
+$Config_PartType_Exercise: (moveTo: $AREA_EOC, resetAt: $RESET_CHAPTER, numberAt: $NUMBER_BEFORE_MOVE, titleContent: $Config_exerciseTitleContent);
+
+$Config_SetTableCaption : (captionType: $CAPTION_TABLE, defaultContainer: caption, hasCaption: true, hasTitle: true, hasTopTitle: true);
+
+$Config_hasCompositeAppendixes: true;
```
Comment (wilkus27) [inline recipes/books/bca/_config.scss:149]: This is old config variable that is not used anymore. It can be removed.

**Review by wilkus27 — CHANGES_REQUESTED** [review]

**Review by wilkus27 — APPROVED** [review]

**Review by brittany-johnson — APPROVED** [review]

**Pull request merged**